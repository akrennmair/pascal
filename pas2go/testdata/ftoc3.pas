program ftoc;

var fgr, cgr : real;
	thirtytwo : integer;
	five : integer;
	nine : integer;

begin
	thirtytwo := 32;
	five := 5;
	nine := 9;
	write('Degree F: ');
	readln(fgr);
	cgr := (fgr-thirtytwo)*five/nine;
	writeln('Degree C: ', cgr:7:2)
end.
