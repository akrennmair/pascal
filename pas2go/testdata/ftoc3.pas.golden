package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program ftoc
func main() {
	var (
		fgr       float64
		cgr       float64
		thirtytwo int
		five      int
		nine      int
	)
	_ = fgr
	_ = cgr
	_ = thirtytwo
	_ = five
	_ = nine

	thirtytwo = 32
	five = 5
	nine = 9
	system.Write("Degree F: ")
	system.Readln(&fgr)
	cgr = (fgr - float64(thirtytwo)) * float64(five) / float64(nine)
	system.Writeln("Degree C: ", cgr)
}
